gis/grass/Portfile

```diff
@@ -43,7 +43,7 @@ depends_lib         port:bzip2 \
                     port:zlib \
                     port:readline
 
-patchfiles          patch-dyld-link.diff
+patchfiles          01_patch-dyld-link.diff 02_grass-python.diff
```

Comment:
My only suggestion for future PRs - and this is a bit of a personal choice, so don't take this as gospel (or required) - is to only apply one patch per line. Likewise, always use `patchfiles-append`.

The benefit is that you can quickly comment/uncomment individual patches during testing, without having to change multiple lines, deal with line continuation, etc.

So this is a better way to do it:

```
patchfiles-append   01_patch-dyld-link.diff
patchfiles-append   02_grass-python.diff
```

No need to change this though, as it's not worth going through another CI cycle.

Reply:
Noted, thanks!